src/main/resources/tags/info/version.tag

```diff
@@ -0,0 +1,10 @@
+tpye: text
+aliases: ver
+
+---
+
+Geyser is built to only support the latest major release of Bedrock at any given time. Betas are not supported, and older version are not supported for maybe more than a minor revision or two for a limited time.
+
+If you need to use Geyser with your own "outdated" server, you need to install [ViaVersion](https://www.spigotmc.org/resources/viaversion.19254/). You must either: use Java 16 on your server; or if your plugins are not compatible, you must either use the standalone or a Bungeecord/Velocity setup running on Java 16.
```

Comment:
```suggestion
If you need to use Geyser with your own "outdated" server, you need to install [ViaVersion](https://www.spigotmc.org/resources/viaversion.19254/). You must use Java 16 on your server. If you cannot run it, we recommend using standalone or Bungeecord/Velocity versions that can run on Java 16.
```
A lot of "either" here makes this hard to flow.

Reply:
Fair point, but I personally don't like the revised wording either... I'll think about it a bit.

Reply:
```You must use Java 16 on your server. If you can't or won't run that, we recommend using standalone or Bungeecord/Velocity versions that can run on Java 16.``` 

maybe?

Reply:
I think that `Geyser requires Java 16 on your server to run.` sounds a bit better than `You must use Java 16 on your server.`